src/Selenium2Library/keywords/_element.py

```diff
@@ -636,7 +639,7 @@ def get_matching_xpath_count(self, xpath):
         `Xpath Should Match X Times`.
         """
         count = len(self._element_find("xpath=" + xpath, False, False))
-        return str(count)
+        return str(count) if return_str else count
```

Comment:
I agree, but would like to do it same time for all the keywords using booleans. There is #719 for this problem.